zebra/rt_netlink.c

```diff
@@ -4350,6 +4364,7 @@ ssize_t netlink_macfdb_update_ctx(struct zebra_dplane_ctx *ctx, void *data,
 	uint32_t update_flags;
 	bool nfy = false;
 	uint8_t nfy_flags = 0;
+	uint32_t nda_ext_flags = 0;
```

Comment:
In this function nda_ext_flags are being set and debugged, but on line 4472 when it is time to encode these flags are not sent to the kernel.  Is this intentional?